src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,77 @@ user_questions:
   - What is the architecture of the Giant Swarm cloud-native developer platform?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-planeteers
-last_review_date: 2024-04-26
+last_review_date: 2024-11-04
 ---
 
-## Platform API {#platform-api}
+Giant Swarm's cloud-native platform is a collection of open-source components that work together to provide a seamless experience for managing the lifecycle of containerized applications. The platform is designed to be cloud-agnostic, allowing you to deploy your applications on any of the supported cloud providers. Besides, the platform offers a rich set of APIs that our customers can use to offer a set of self-service capabilities to their developers.
+
+## Platform architecture
+
+![Platform architecture](./platform-architecture.png)
+
+The platform architecture consists of the following layers:
+
+- Interface layer: offers a set of templates, APIs, and tools that developers can use to interact with the platform.
+
+- Capabilities layer: provides a set of capabilities to manage the lifecycle of applications, including deploying, scaling, and monitoring applications.
+
+- Infrastructure layer: responsible for managing the underlying infrastructure, including the cloud provider resources or on-premises infrastructure. To abstract the underlying infrastructure, the platform uses the `Kubernetes`.
+
+Along these layers, Giant Swarm provides a real support channel, incident management, and good documentation to help you get the most out of the platform. Additionally, you have a dedicated account engineer to help creating a common roadmap and to provide guidance on how to use the platform.
+
+## Platform API
+
+The Giant Swarm platform is build on top of `Kubernetes` thanks to [`Cluster API`](https://cluster-api.sigs.k8s.io/), a open source  project that standardize the cluster lifecycle management across different cloud providers or on-premises infrastructure.
```

Comment:
```suggestion
The Giant Swarm platform is build on top of `Kubernetes` based on [`Cluster API`](https://cluster-api.sigs.k8s.io/), an open source project that standardizes the cluster lifecycle management across different cloud providers or on-premises infrastructure.
```

Reply:
I would start by mentioning that the platform API that is exposed is the kubernetes api. That all the exising tooling works with our platform api. Examples: GitOps, kubectl even ci/cd systems/terraform etc. Also the authentication and authorization is the same.

And then go into the topology. There we also need to mention that the management cluster (and the platform api) is not a saas by giantswarm. But instead deployed, maintained and operated by giant swarm within the premises of the customer.

Reply:
+1. And: please don't quote Kubernetes or other terms (CAPI, Git, ...) as code block with backticks. Those terms are well-known and don't need formatting (no italics, no quoting, just a link and spelling out the abbreviation at first use is enough). We should mention CAPI as _subproject_ of Kubernetes to denote it as close neighbor.